### DISCUSSION TITLE
Domotica casa nuova in costruzione

### DISCUSSION THREAD

**Discussion by antony84** [indomus/forum#2157] [Hardware & Firmware]
Ciao a tutti,
mi sono documentato un po' e vorrei proporre al mio elettricista (che sta per entrare in cantiere, trattasi di villetta a due piani) di usare gli shelly serie PRO (da quadro e rete Lan) per domotizzare luci esterne e interne (quasi tutte), alcune prese, irrigazione, cancello e portone ingresso.
Oltre al comando dei singoli dispositivi, vorrei poter creare anche dei gruppi e degli scenari, sempre con una sola app (per non dover fare le cose più volte in più app), ed alcuni di questi gruppi comandarli anche tramite assistente vocale, preferibilmente Alexa. Per fare ciò l'idea è usare home assistant.

Credete sia la strada giusta da percorrere? O sarebbe meglio orientarsi su altro?

**Comment by snakuzzo** [comment]
strada giusta, ma inutile aprire l'ennesima discussione su questo argomento. Se cerchi tra le discussioni ne troverai sicuramente altre.
Chiudo
